assets/routes.jsx

```diff
@@ -15,9 +15,9 @@ export default (store, constants) => {
   return <Provider store={store}>
```

Comment:
By the way, you were right that manually passing urls down to the Wrapper is a bad idea. Now, thanks to React Context we can access the route information (including url) directly from the Wrapper. ReactRouter supports React Context api, so all we need to do is to use `withRouter` HOC. Please remove all changes from pages and routes.jsx, and access url directly from Wrapper.

https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/withRouter.md